lib/features/personal_summary/overview/dialogs/first_use_summary_dialog.dart

```diff
@@ -24,6 +24,13 @@ class _FirstUseSummaryDialogState extends State<FirstUseSummaryDialog> {
   int currentIndex = 0;
   final CarouselSliderController _controller = CarouselSliderController();
 
+  void permanentlyDismiss() {
```

Comment:
Rather than have this code here and needing it for close button, other buttons, back button etc I would move it to the void _showFirstUseDialogs(BuildContext context) method. Because that's all your checking for, whether you have shown them, not wether you have shown them and the user dismissed them. Would clean this file up a bit and keep the logic in one place.

Reply:
uu awesome idea